includes/wikia/tests/HeliosHelperControllerTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+use Wikia\Helios\HelperController;
+
+class HeliosHelperControllerTest extends WikiaBaseTest {
+
+	public function test_shouldFailWithInvalidSecret() {
+		$response = $this->app->sendRequest( 'Wikia\Helios\HelperController', 'sendPasswordResetLinkEmail', [] );
+		$this->assertEquals( \WikiaResponse::RESPONSE_CODE_FORBIDDEN, $response->getCode() );
+		$this->assertFalse( $response->getData()['success'] );
+		$this->assertEquals( 'invalid secret', $response->getData()['message'] );
+	}
+
+	public function test_shouldFailInvalidToken() {
+		global $wgTheSchwartzSecretToken;
+		$response = $this->app->sendRequest( 'Wikia\Helios\HelperController', 'sendPasswordResetLinkEmail', [
+			HelperController::SCHWARTZ_PARAM => $wgTheSchwartzSecretToken,
+		] );
+
+		$this->assertEquals( \WikiaResponse::RESPONSE_CODE_BAD_REQUEST, $response->getCode() );
+		$this->assertFalse( $response->getData()['success'] );
+		$this->assertEquals( HelperController::ERR_INVALID_TOKEN, $response->getData()['message'] );
+
+		$response = $this->app->sendRequest( 'Wikia\Helios\HelperController', 'sendPasswordResetLinkEmail', [
+			HelperController::SCHWARTZ_PARAM => $wgTheSchwartzSecretToken,
+			'user_id'                        => 4,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user_id" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)